### DISCUSSION TITLE
Please tell me your favorite singer!

### DISCUSSION THREAD

**Discussion by kanda-shogo** [NAIST-SE/AttractingContributors#173] [General]
https://github.com/kanda-shogo/AttractingContributors-singer/blob/main/README.md

**Comment by yangziwei96** [comment]
added! and plz help me too
https://github.com/yangziwei96/AttractingContributors-sports

**Comment by ElefunAi** [comment]
I have added singer list!
Please check and help me!

https://github.com/ElefunAi/AttractingContributors-texteditor

**Comment by t-matsumoto4758** [comment]
Added!
please check and help me!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Comment by Gavin-23** [comment]
Added!
Please help me too~~
Thanks 😊
https://github.com/Gavin-23/AttractingContributors-Music
